### PR TITLE
Fix email layout in the Employees info.

### PR DESCRIPTION
**Pull request by e-reeley**
Within the employees information view, the emails are joined on the same line, although they are seperate addresses.

## Description

This change allows the emails to be displayed on their own line.

## How To Test This

1) Navigate to the Employees module
2) Select one of the to view the details
3) View the Email addresses at the bottom
4) They should all be on seperate lines.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by Dillon-Brown** [comment]
Assessed

**Comment by zoltankocsardi** [comment]
It works!
code +1

**Comment by Dillon-Brown** [comment]
Assessed

File: themes/SuiteP/css/suitep-base/panels.scss [hunk 1]
```diff
@@ -115,7 +116,7 @@
   font-family: 'Glyphicons Halflings';
   content: "\2212";
   font-weight: 100;
-  line-height: 40px/$panel-height-divisor;
+  //line-height: 40px/$panel-height-divisor;
 }
```
Comment (PedroErnst) [inline themes/SuiteP/css/suitep-base/panels.scss]: Why not remove this line?

**Pull request merged**